packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -150,6 +150,10 @@ async function generateBase64({ file, args, reporter }) {
     return null
   }
 
+  if (options.trim) {
+    pipeline = (typeof options.trim === 'number') ? pipeline.trim(options.trim) : pipeline.trim();
```

Comment:
The type check isn't needed if using `GraphQLInt`, but I left it in case someone had a better suggestion.

I also considered having separate `trim` (as `GraphQLBoolean`) and `trimThreshold` and trying to coalesce them into a single `trim` option, but that seemed unnecessarily complicated just to retain a default of `10`.